File: C# Pattern Programs/Alphabetic Patterns/Pattern 91.cs
using System;

class Pattern_227
{
    public static void Main()
    {
        string str = "ABCDE";
        for (int i = 0; i < str.Length; i++)
        {
            for (int j = 0; j < str.Length; j++)
            {
                if (i == 2)
                {
                    Console.Write(str[j]);
                }
                else if (j == 2)
                {
                    Console.Write(str[i]);
                }
                else
                {
                    Console.Write(" ");
                }
            }
            Console.WriteLine("");
        }
        Console.ReadKey(true);
    }
}

File: PythonPatternPrograms/Alphabetic Patterns/Pattern 91.py
string = "ABCDE"

for x in range(0, 5):
 for y in range(0, 5):
  if x == 2:
   print(string[y],end=" ")
  elif y == 2:
   print(string[x],end=" ")
  else:
   print(" ",end=" ")
 print("")